pkg/security/api/types.go

```diff
@@ -79,22 +79,22 @@ type PodSecurityPolicyReview struct {
 
 // PodSecurityPolicyReviewSpec defines specification for PodSecurityPolicyReview
 type PodSecurityPolicyReviewSpec struct {
-	// PodSpec is the PodSpec to check. The PodSpec.ServiceAccountName field is used
-	// if ServiceAccountNames is empty, unless the PodSpec.ServiceAccountName is empty,
+	// Template is the PodTemplateSpec to check. The PodTemplateSpec.Spec.ServiceAccountName field is used
+	// if ServiceAccountNames is empty, unless the PodTemplateSpec.Spec.ServiceAccountName is empty,
 	// in which case "default" is used.
-	// If ServiceAccountNames is specified, PodSpec.ServiceAccountName is ignored.
-	PodSpec kapi.PodSpec
+	// If ServiceAccountNames is specified, PodTemplateSpec.Spec.ServiceAccountName is ignored.
+	Template kapi.PodTemplateSpec
```

Comment:
Name change confirmed.  If ServiceAccountNames nil vs specified? Or empty is allowed?